### PR TITLE
Improving nav bar

### PR DESCRIPTION
**Pull request by DoctorDib**
Changes:
 - Added sticky nav bar feature for easier access on all pages.

### CODE REVIEW

File: client/src/Components/Header/index.tsx [hunk 1]
```diff
@@ -22,7 +22,14 @@ const LogoButton = withStyles({
 const DesktopHeader: React.FC = () => {
     const classes: IClasses = styles();
 
-    return (
+    const [scrolled, setScrolled] = React.useState(false);
+    const handleScroll = () => setScrolled(window.scrollY > 100);
+
+    useEffect(() => {
+        window.addEventListener('scroll', handleScroll);
```
Comment (BenRGH) [inline client/src/Components/Header/index.tsx:29]: You need to add a return statement that removes the listener or we get memory leaks

Comment (BenRGH) [reply client/src/Components/Header/index.tsx:29]: Also add an empty array as the second arg for the useEffect or it'll rerun every render

Comment (DoctorDib) [reply client/src/Components/Header/index.tsx:29]: I think your first comment is related to the second. I did forget to add the empty array which I have just changed

Comment (BenRGH) [reply client/src/Components/Header/index.tsx:29]: Nah you still need the return https://stackoverflow.com/a/55566585/9982232

Comment (DoctorDib) [reply client/src/Components/Header/index.tsx:29]: https://codedaily.io/tutorials/72/Creating-a-Reusable-Window-Event-Listener-Hook-with-useEffect-and-useCallback

**Review by BenRGH — APPROVED** [review]

**Pull request merged**